workflows/s3_object_lock/README.md

```diff
@@ -0,0 +1,39 @@
+# Amazon S3 Object Lock Workflow
+
+## Overview
+
+This example shows how to use AWS SDKs to work with Amazon Simple Storage Service (Amazon S3) object locking features. The workflow demonstrates how to create, update, view, and modify object locks, as well as how locked objects behave regarding requests to delete and overwrite.
+
+[Amazon S3 Object Lock](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lock.html) can help prevent Amazon S3 objects from being deleted or overwritten for a fixed amount of time or indefinitely. Object Lock can help meet regulatory requirements or protect against object changes or deletion.
+
+![Object Lock Features](resources/Diagram_Amazon-S3-Object-Lock.png)
+
+This workflow demonstrates the following steps and tasks:
+1. Add object lock settings to S3 buckets, to new and existing buckets.
```

Comment:
(Opt) Add object lock settings to S3 buckets for both new and existing buckets.

Reply:
Fixed to match above.